docs/access-management/rbac.md

```diff
@@ -0,0 +1,54 @@
+---
+products: oss-enterprise, cloud-teams
+---
+
+# Role Based Access Control (RBAC)
+
+Role Based Access Control allows a user with Administrative access to apply roles to users, granting different levels of permission within an organization or workspace. 
+
+## Workspace Resource Roles
+A user can have at most one role of this kind per workspace. Permissions are scoped to the specific workspace in which the user has this role.
+
+| Permissions           | WorkspaceReader    | WorkspaceEditor    | WorkspaceAdmin    |
```

Comment:
Can we please use the name of the permissions in the UI, rather than the name of the permissions in our code?

Reply:
I can see within the workspace UI, that the roles are Reader, Editor, and Admin. I don't have Org view access. Are those roles also shortened in the UI? "Member", "Reader", "Editor", "Admin"? Or do org role names show as "Organization Admin" etc? Because I see this in the workspace settings, my guess is that Org level role names include "Organization"
![Screenshot 2024-06-17 at 11 21 40 AM](https://github.com/airbytehq/airbyte/assets/23410478/7429420e-ac3f-430e-affc-185dec1b59de)